source/documentation/tools/quicksight/04-working-with-quicksight.md

```diff
@@ -0,0 +1,44 @@
+# Working with QuickSight
+
+## Finding datasets
```

Comment:
It looks like 'Add a dataset' or 'Add datasets' might be more accurate for what this section covers :)